src/codeu/chat/server/Server.java

```diff
@@ -221,6 +221,67 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
             }
         });
 
+        //User Access Level- A client wants to get the user's access level for a conversation
+        this.commands.put(NetworkCode.NEW_ACCESS_LEVEL_REQUEST, new Command() {
+          @Override
+          public void onMessage(InputStream in, OutputStream out) throws IOException {
+            final Uuid conversationId = (Uuid.SERIALIZER).read(in);
+            final Uuid userId = (Uuid.SERIALIZER).read(in);
+            final AccessLevel accessLevel = AccessLevel.valueOf(Serializers.STRING.read(in));
+            Serializers.INTEGER.write(out, NetworkCode.NEW_ACCESS_LEVEL_RESPONSE);
+            final UserAccessLevel userAccess = controller.newUserAccessLevel(conversationId, userId, accessLevel);
+            Serializers.nullable(UserAccessLevel.SERIALIZER).write(out, userAccess);
+
+//fix
+            logQueue.getTransactions().add("User Access Level: " + userId.toString() + " " + accessLevel.toString() + " " );
```

Comment:
`"User Access Level: "` doesn't look right to me.

Never mind the inconsistency (other commands are named in `ALL-CAPS-WITH-HYPHENS`), but I don't see code in `readTransactionLog()` that knows how to read this command back.

So, I'm guessing that persistence of user access level setting is busted right now, right?